sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -456,13 +456,16 @@ trait V2CreateTablePlan extends LogicalPlan {
 
 /**
  * Create a new table with a v2 catalog.
+ * The [[defaults]] hold optional default value expressions to use when creating the table,
+ * mapping 1:1 with the fields in [[tableSchema]].
  */
 case class CreateTable(
     name: LogicalPlan,
     tableSchema: StructType,
     partitioning: Seq[Transform],
     tableSpec: TableSpecBase,
-    ignoreIfExists: Boolean)
+    ignoreIfExists: Boolean,
+    defaults: Seq[Option[Expression]])
```

Comment:
Could you rename `defaults` to `defaultValues` or `defaultValueExpressions` here? `default` looks a little too broad in the context of `Table` definition.

Reply:
Sure, this is done.